action.yml

```diff
@@ -43,7 +43,7 @@ runs:
   using: 'composite'
   steps:
   - name: Solsta setup
-    uses: snxd/deploy-github-setup-action@v2
+    uses: snxd/deploy-github-setup-action@D4T-570_fix
```

Comment:
Is this right? It seems odd to use a tag for a specific fix.

Reply:
No it's not.  I missed this -- the readme is correct, but the action is wrong.  Fixing.  I just saw the top and assumed this was the commit that should have gone in last week... ugh.